frontend/sw360-portlet/src/main/webapp/html/projects/detail.jsp

```diff
@@ -44,7 +44,11 @@
     <div id="header"></div>
     <p class="pageHeader"><span class="pageHeaderBigSpan">Project: <sw360:ProjectName project="${project}"/></span>
         <span class="pull-right">
-        <input type="button" id="edit" value="Edit" class="addButton">
+        <input type="button" id="edit" value="Edit" class="addButton"
+            <c:if test = "${(project.clearingState eq 'CLOSED') && (isUserAdmin != 'Yes')}">
++                 <c:out value="disabled='disabled'"/>
```

Comment:
```suggestion
                 <c:out value="disabled='disabled'"/>
```

Reply:
Still a Problem

Reply:
I could not clearly understand the review comment completely here.If understood is correctly your concern is about having this "disabled" property. I have kept the feature for two reasons.
1.User will be able to immediately  know that he is not allowed to edit the project and he does not have to wait until  the response comes from backend.
2.This willl reduce the number of backend calls.
And you might already know even if user enable the "Edit" button from the browser then also user will not be able to actually edit because of the backend validation.

Reply:
This comment is only about the additional `+` at the beginning of the line.

Reply:
Corrected the indentation.